Client/Frontend/Browser/SearchViewController.swift

```diff
@@ -276,6 +262,15 @@ class SearchViewController: SiteTableViewController, KeyboardHelperDelegate, Loa
             reloadData()
         }
     }
+    
+    // Ecosia: Unleash Shortcuts Experiment
+    var tableViewBottomEqualToAnchor: NSLayoutYAxisAnchor {
+        if EngineShortcutsExperiment.isEnabled {
```

Comment:
Without deeper knowledge of the rest of the code: But is this code considering that the toolbar can be at top and bottom?

Reply:
This is the anchor that is being added at the TableView's bottom anchor.
In the case of the `EngineShortcutsExperiment.isEnabled` the anchor isthe toolbar so it can be shown, ViewController's view otherwise.

Reply:
Jep, I see it works. Quickly cross-checked by running the app.